src/components/SlidingPane/SlidingPane.scss

```diff
@@ -0,0 +1,52 @@
+@import '~om';
+
+.SlidingPane {
+  $siteNavOverlay: 9000;
```

Comment:
I think this doesn't belong to `om`, let's make this have `z-index: 1` and we can override the z-index in discovery/any other apps we use this in.

Reply:
I think this is OK, but instead of using 9000, we should give this a name and expose it.  The problem with overriding at the call site is that it's not obvious that someone needs to do that, and it's likely that the component won't work quite right out of the box.

Reply:
We can override it in [globals.scss](https://github.com/ModaOperandi/discovery/blob/master/src/global.scss#L8-L14) in discovery similar to how we do it for Modal and Overlay.
The problem with defining the z-index in om is that not all apps use the same z-index for overlays, modals, etc.

Reply:
Although we could standardize them across repos and just add the z-indexes to [tokens](https://github.com/ModaOperandi/tokens)?

Reply:
I like that idea.  Let's roll with it.

Reply:
Might be a good idea to create a separate ticket for adding z-indexes to tokens. I believe there'll be more than one repo involved in this 🤔